Fixed the bugs I found and made it responsive in Brutalist Webpage.

fabcodingzest opened the pull request:
I used some flexbox to manage the sections and made it responsive a bit.
Do you want any other changes? :)

fabcodingzest committed:
Fixed the bugs I found and made it responsive a bit

fabcodingzest committed:
Fixed the bugs I found and made it responsive a bit for Brutalist Webpage

jameesy commented on Portfolios/Brutalist/style.css:
```diff
@@ -7,107 +6,118 @@ FLEXBOX
 body {
-  margin-left: 60px;
-  margin-top: 20px;
+	margin-left: 60px;
```
Hey there,

Thanks for your contribution :)

I think what has happened here is that your code formatter has changed the tab rules of the file. If you could recommit with the repo defaults, the changes displayed will be limited to just the important bits! 

Many thanks :)

fabrizvi replied:
Okay, I will do that.

fabcodingzest replied:
Hey!
Okay so I am totally new to the PR thing but I tried to make some changes and I don't know what to do from here. I have made some more commits, could you please check and tell me if Tabs space is still a problem? I don't know how to do what you asked and made a mess ugh. I am so sorry :(

jameesy reviewed (changes requested):
If you could address the comment I left then that would be awesome! <3

fabcodingzest committed:
removed the autoprefixers

fabcodingzest committed:
Fixed the tab space problem

fabcodingzest committed:
Fixed the tab space in brutialist css file

fabcodingzest committed:
Update style.css

fabcodingzest committed:
added the tools class again

um, I don't know this PR stuff so while updating - the tools class and other changes were lost, hopefully this will resolve it.

fabcodingzest committed:
minor bug fiexing

fabcodingzest committed:
minor bug fixing

fabcodingzest committed:
Merge branch 'minor-bug-fixes' of github.com:fabrizvi/Dev-Folio into minor-bug-fixes

fabcodingzest committed:
Revert "Merge branch 'minor-bug-fixes' of github.com:fabrizvi/Dev-Folio into minor-bug-fixes"

This reverts commit 4db4c7a9284e4ff1b8f88d26ed82678a51369277, reversing
changes made to 52c83a15baad7470ad418436d393b11d78bb6470.

fabcodingzest committed:
header fixed

fabcodingzest committed:
tried to fix tab spaces
